From 30df9454d901fe6cb75028f4868439e01fe8f521 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20Van=C4=9Bk?= <arkamar@gentoo.org>
Date: Wed, 26 Feb 2025 08:39:31 +0000
Subject: [PATCH] Hard disable Redis and MongoDB datastore plugins

This solves automagic properties of those two plugins.

See-also: https://github.com/sysrepo/sysrepo/pull/3554
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4992e29c..eb4b3594 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -387,7 +387,7 @@ list(REMOVE_ITEM CMAKE_REQUIRED_DEFINITIONS -D_DEFAULT_SOURCE)
 # libmongoc - optional
 find_package(mongoc-1.0 1.24.0 CONFIG)
 find_program(MONGOSH mongosh)
-if(TARGET mongo::mongoc_shared AND MONGOSH)
+if(0 AND TARGET mongo::mongoc_shared AND MONGOSH)
     # datastore plugin added if libraries exist
     list(APPEND LIB_SRC src/plugins/ds_mongo.c)
     set(SR_ENABLED_DS_PLG_MONGO 1)
@@ -414,7 +414,7 @@ endif()
 # libhiredis - optional
 find_package(LibHiredis 1.1.0)
 find_program(REDIS_CLI redis-cli)
-if(LIBHIREDIS_FOUND AND REDIS_CLI)
+if(0 AND LIBHIREDIS_FOUND AND REDIS_CLI)
     # datastore plugin added if libraries exist
     list(APPEND LIB_SRC src/plugins/ds_redis.c)
     set(SR_ENABLED_DS_PLG_REDIS 1)
-- 
2.48.1

